homeassistant/components/tuya/strings.json

```diff
@@ -14,12 +14,12 @@
       }
     },
     "abort": {
-      "auth_failed": "[%key:common::config_flow::error::invalid_auth%]",
-      "conn_error": "[%key:common::config_flow::error::cannot_connect%]",
+      "invalid_auth": "[%key:common::config_flow::error::invalid_auth%]",
+      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]",
```

Comment:
Isn't it bad to use error keys for aborts? Although it is not subject of change here.

Reply:
No, that's ok.